.pa11yci

```diff
@@ -0,0 +1,10 @@
+{
+    "defaults": {
+        "ignore": [
+            "WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail"
```

Comment:
Do we want to ignore contrast-related errors on cdh-web as well as for PPA? I turned this off because a particular PPA color combination was just below the threshold, but that will probably have changed with the new PPA design, and it might be useful to know if cdh-web has contrast problems.